pkg/intermediate/aggregate_test.go

```diff
@@ -266,18 +271,17 @@ func TestCorrelateRecords(t *testing.T) {
 	flowKey1, _ := getFlowKeyFromRecord(record1)
 	record2 := createMsgwithDataSet2().GetSet().GetRecords()[0]
 	flowKey2, _ := getFlowKeyFromRecord(record2)
-	aggregationProcess.correlateRecords(*flowKey1, record1)
-	aggregationProcess.correlateRecords(*flowKey2, record2)
+	aggregationProcess.aggregateRecord(*flowKey1, record1)
+	aggregationProcess.aggregateRecord(*flowKey2, record2)
```

Comment:
The logic problem when correlating ipv4 fields is not reflected in the unit tests. I think we should add a case that reverse the sequence of sending these two records, which will cover the case that destination record is received first.

Reply:
Yes, there was a typo for the case of uint and ints in L187, it carried over. 
We are missing that scenario. In addition, we are also missing tests for intra-node flows. Was planning to add them.